org_fedora_hello_world/ks/hello_world.py

```diff
@@ -154,7 +154,7 @@ def setup(self, storage, ksdata, instclass, payload):
         # no actions needed in this addon
         pass
 
-    def execute(self, storage, ksdata, instclass, users, payload):
+    def execute(self, storage, ksdata, instclass, users):
```

Comment:
So addons can no longer modify the package payload???

Reply:
well, from Fedora 30 this error was showing - `__init__() missing 1 required positional argument payload`. So, removing payload fixed it.

Reply:
I haven't noticed that. Please, remove only the argument `instclass`.

Reply:
That error pops up after clicking on **Begin Installation** button. Of course, I may add payload if you would still like to.

Reply:
I should have been more specific. Please, remove the argument `instclass` instead of `payload` in the methods `setup` and `execute`. You removed a wrong argument, that is all :-)

Reply:
Ok! changed. Thanks for your guidance!

Reply:
I got it now. Removing payload equals the number of arguments. So, it fixed the error in my case. Thanks!